jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -236,12 +235,14 @@ YUI.add('user-profile', function() {
       @method switchModel
       @param {String} uuid The model UUID.
       @param {String} name The model name.
+      @param {Function} callback The function to be called once the model has
```

Comment:
Please describe what's passed to the callback (the env object if I understand this correctly).